java/websvc.saas.codegen.java/src/org/netbeans/modules/websvc/saas/codegen/java/support/SourceGroupSupport.java

```diff
@@ -184,7 +184,7 @@ private static Map createFoldersToSourceGroupsMap(final SourceGroup[] sourceGrou
     private static List<SourceGroup> getTestTargets(SourceGroup sourceGroup, Map foldersToSourceGroupsMap) {
         final URL[] rootURLs = UnitTestForSourceQuery.findUnitTests(sourceGroup.getRootFolder());
         if (rootURLs.length == 0) {
-            return new ArrayList();
+            return new ArrayList<SourceGroup>();
         }
         List<SourceGroup> result = new ArrayList<SourceGroup>();
```

Comment:
Also I know that probably it was not a target, but diamond here?

Reply:
No problem.. Done..